src/Raven.Server/Documents/Indexes/Auto/AutoMapIndexDefinition.cs

```diff
@@ -127,21 +127,22 @@ public static AutoMapIndexDefinition LoadFromJson(BlittableJsonReaderObject read
                 throw new InvalidOperationException("No persisted lock mode");
 
             var fields = new AutoIndexField[jsonArray.Length];
-
             for (var i = 0; i < jsonArray.Length; i++)
             {
                 var json = jsonArray.GetByIndex<BlittableJsonReaderObject>(i);
 
                 json.TryGet(nameof(AutoIndexField.Name), out string name);
                 json.TryGet(nameof(AutoIndexField.Indexing), out string indexing);
                 json.TryGet(nameof(AutoIndexField.HasSuggestions), out bool hasSuggestions);
+                json.TryGet(nameof(AutoIndexField.Id), out int id);
```

Comment:
Let's assume it's an old index (built by Lucene) and we don't have the id prop in JSON
Would TryGet throw an exception? If not we don't care because Lucene isn't using Id anyways.

Reply:
No, it will return `false` and set the `id` to `0`

Reply:
So, should we care?

Reply:
Not except to note that we aren't storing the engine type

Reply:
The engine type is saved as a persistent field like analyzators etc.